### DISCUSSION TITLE
Server DataStorage 사용

### DISCUSSION THREAD

**Discussion by soontae1** [naverz/zepeto-studio-kor#282] [5. Q&A]
일전에 외부 통신을 사용하여 로그 데이터를 수집하지 못한다고 하여 Server DataStorage를 사용하려고 합니다

게임상에서 저장된 DataStorage 데이터를 외부에서 조회 할 수 있는 방법이 있을까요?

**Comment by hyeonjin-jo** [comment]
안녕하세요.

Server DataStorage에 저장된 데이터는 외부에서 접근이 불가능합니다.

**Reply by soontae1** [reply]
제작하려는 플랫폼 특성상 로그데이터를 외부에서 확인 하였으면 하는데
앞으로도 계획은 없는건가요?

**Reply by hyeonjin-jo** [reply]
로그 데이터를 수집 및 조회하는 기능과 Server DataStorage에 저장된 데이터를 조회하는 기능 모두 내부에서 검토 중이라서
이후에는 오픈될 가능성이 있으나 정확한 일정을 공유드리긴 어려운 점 양해 부탁드립니다.

**Reply by gibbeum-lee** [reply]
@hyeonjin-jo 로그 데이터 수집과 Server DataStorage가 꼭 필요한데, 내부 논의가 어떻게 진행되셨는지 궁금합니다.
